UltimateTeam.Toolkit/Models/AuctionResponse.cs

```diff
@@ -2,13 +2,13 @@
 
 namespace UltimateTeam.Toolkit.Models
 {
-    public class AuctionResponse
+    public class AuctionResponse : IListResponse
```

Comment:
I confess I do not remember why I did this. Lol
It must have been some crazy thing that I started to do, but I gave up. It really does not make much sense for the methods to be virtual in this case.